### ISSUE TITLE
loss output: is this normal?

### ISSUE THREAD

**Issue by askerlee** [Eniac-Xie/faster-rcnn-resnet#16]
Hi, I'm using it on my own dataset. Some of the output losses are always 0, which seems not the case when I used py-faster-rcnn. Could you please inform if there are misconfigurations? Thank you so much.

```
I1019 22:34:34.436921 14581 solver.cpp:229] Iteration 760, loss = 0.0427504
I1019 22:34:34.436954 14581 solver.cpp:245]     Train net output #0: loss_bbox = 0 (* 1 = 0 loss)
I1019 22:34:34.436959 14581 solver.cpp:245]     Train net output #1: loss_cls = 0 (* 1 = 0 loss)
I1019 22:34:34.436962 14581 solver.cpp:245]     Train net output #2: rpn_cls_loss = 0.0208707 (* 1 = 0.0208707 loss)
I1019 22:34:34.436965 14581 solver.cpp:245]     Train net output #3: rpn_loss_bbox = 0.00372629 (* 1 = 0.00372629 loss)
```

**Comment by whmin** [comment]
@askerlee @Eniac-Xie Could you please release your "ResNet101_BN_SCALE_Merged_OHEM" model files included test.prototxt,my download files not contain it,but i have no time to write it because of an emergency.Thank you very much!!!

**Comment by askerlee** [comment]
Just copy the test.prototxt from the ResNet101_BN_SCALE_Merged folder. They are the same (hard example mining only happens in training, so the test model is the same).

**Comment by whmin** [comment]
Ok,thank you!!!Now i got an error when run  ./experiments/scripts/faster_rcnn_end2end.sh 0 ResNet-50 pascal_voc,like this:
![screenshot from 2017-11-08 21-08-43](https://user-images.githubusercontent.com/23375545/32589099-8ecf7b70-c54d-11e7-9a0e-d974514fb230.png)
i did not change the original code about resnet-50 with ohem,but only replace the "num_classes" and "num_output",i can not solve it,could you help me?

**Comment by askerlee** [comment]
change cls_prob[i,label] to cls_prob[i,int(label)] in lib/roi_data_layer/layer.py:242.

**Comment by oysz2016** [comment]
Hi, I have encountered the same problem. Have you solved it?I think it may be the problem of the code. When I used the OHEM code modified by myself to train the author's prototxt file, the loss was not 0, but it was difficult to converge (which did not exist on VGG16).
